### ISSUE TITLE
[Bug] Dropdown position bugged since 5.0.0

### ISSUE THREAD

**Issue by JoniJnm** [mdbootstrap/mdb-angular-ui-kit#163]
Dropdown is bugged since new [angular version 5.0.0](https://mdbootstrap.com/docs/angular/getting-started/changelog/#v5-0-0)

Steps to reproduce:

```shell
git clone git@git.mdbootstrap.com:mdb/angular/mdb5/prd/mdb5-angular-ui-kit-pro-advanced.git
cd mdb5-angular-ui-kit-pro-advanced/templates/mdb-angular-admin-dashboards
npm install
npm start
```

Open user menu, the position is wrong

Tested in:
* Chrome 114.0.5735.198
* Firefox 115.0

![image](https://github.com/mdbootstrap/mdb-angular-ui-kit/assets/577282/8a862dfb-022d-44e7-8494-bc7041d21597)

**Comment by idzark** [comment]
@MartinSchmidtStudyly If there are any problems with Popover and Select components, please consider creating a new issue with more detailed information about the bug and reproduction steps.

**Comment by idzark** [comment]
The problem with dropdown position was fixed in v5.1.0.
